Makefile

```diff
@@ -68,6 +68,15 @@ data.local: export EREGS_PASSWORD=UkOAsfkItN
 data.%: tools/ecfr-parser/build/ecfr-parser
 	./tools/ecfr-parser/build/ecfr-parser -title 42 -subchapter IV-C -parts 400,457,460 -eregs-url $(CORE_URL)
 
+tools/guidance_pipeline/build/guidance_pipeline: tools/guidance_pipeline/*.go
+	cd tools/guidance_pipeline; go build -o build/ .
+
+tools/guidance_pipeline/guidance/*.json: tools/guidance_pipeline/build/guidance_pipeline
```

Comment:
the tool wont work if the folder doesn't exist yet. The makefile could take care of creating the directory if it doesn't exist.